### PR TITLE
Update CodeFund -> EthicalAds

### PR DESCRIPTION
**Pull request by ericholscher**
We're doing what CodeFund used to do: https://www.ethicalads.io/blog/2020/08/announcing-ethicalads/

I'd love to see the blog post updated with this as well, if possible: https://medium.com/open-collective/using-ads-to-sustain-open-source-d048b75d4979

### CODE REVIEW

File: collectives/funding-options.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ Check with your [Fiscal Host](../fiscal-hosts/fiscal-hosts.md) to see if these o
 
 ## Ethical Advertising
 
-Speaking to niche audiences, like developers, in their own environment is one way OSS can leverage the tangible value it provides companies. Examples include CodeFund, Carbon Ads or referral fee solutions like Triplebyte.
+Speaking to niche audiences, like developers, in their own environment is one way OSS can leverage the tangible value it provides companies. Examples include [EthicalAds](https://www.ethicalads.io/), Carbon Ads, or referral fee solutions like Triplebyte.
```
Comment (ericholscher) [inline collectives/funding-options.md:13]: Not sure if the Oxford comma is correct here :)

**Comment by BenJam** [comment]
I just merged in the copy change in GitBook — not sure whether it supports merging straight from git, and I'd rather not risk it as I couldn't find the docs for it 🙃

@piamancini would you be able to add a disclaimer to your medium post linked above? 

> As of Aug 20020 CodeSponsor is now part of [EthicalAds](https://www.ethicalads.io/).

**Review by SudharakaP — COMMENTED** [review]
@BenJam : We are only including the link to EthicalAds. Maybe we should either include the links to the other two or remove the link to EthicalAds just to make it consistent. Just my two cents. 😉

**Comment by ericholscher** [comment]
Thanks for the follow up. I'm fine linking all of them, or none of them. I saw linking to people varied in the different sections, so wasn't sure what was correct.

**Comment by BenJam** [comment]
added 👍

**Pull request closed without merging**